lib/charms/observability_libs/v0/metrics_endpoint_discovery.py

```diff
@@ -126,35 +127,40 @@ def __init__(self, charm: CharmBase, labels: Dict[str, Iterable]):
         self._observer_pid = 0
 
         self._labels = labels
+        self.start_observer()
 
     def start_observer(self):
```

Comment:
should this become private, now that we're calling it from init? Or is there still a reason/way to call this from somewhere else (external)?

Reply:
Since it was previously visible and there's not real data mutation from leaving public, I didn't see a need to make it private. I don't actually think anyone is using it currently (or the public method), but still...